src/integrations/stytch.ts

```diff
@@ -20,7 +20,12 @@ export function installStytch(stytchConfig: any) {
       if (authState.client) resolve(authState.client);
       else {
         const initializedClient = new StytchHeadlessClient(
-          stytchConfig.metadata.token
+          stytchConfig.metadata.token,
+          {
+            cookieOptions: {
+              availableToSubdomains: true
```

Comment:
should this always be true now, or do we add a checkbox to the integration settings?